Nos/Service/CurrentUser.swift

```diff
@@ -99,10 +99,10 @@ class CurrentUser: NSObject, ObservableObject, NSFetchedResultsControllerDelegat
     private var authorWatcher: NSFetchedResultsController<Author>?
 
     @MainActor override init() {
+        self.socialGraph = SocialGraphCache(userKey: nil, context: backgroundContext)
```

Comment:
This line crashes on launch for me because `backgroundContext` is force-unwrapped. In fact, I'm not sure why the tests didn't crash on this line too 🤔